src/languageservice/utils/indentationGuesser.ts

```diff
@@ -0,0 +1,224 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Microsoft? copy-pasted from elsewhere? Maybe worth adding a link to where it comes from to ease maintenance